cdap-docs/build.sh

```diff
@@ -30,6 +30,34 @@ ARG_1="${1}"
 ARG_2="${2}"
 ARG_3="${3}"
 
+function usage() {
+  cd ${PROJECT_PATH}
+  PROJECT_PATH=`pwd`
+  echo "Build script for '${PROJECT_CAPS}' docs"
```

Comment:
Fair enough. We'll keep it as-is until we revamp for the new build after 3.2 is released, then we'll port that work to the other projects.